go/systests/home_test.go

```diff
@@ -87,18 +87,25 @@ func findFollowerInHome(t *testing.T, home keybase1.HomeScreen, f string) (prese
 		if err != nil {
 			t.Fatal(err)
 		}
-		if typ == keybase1.HomeScreenItemType_PEOPLE {
-			people := item.Data.People()
-			typ, err := people.T()
-			if err != nil {
-				t.Fatal(err)
+		if typ != keybase1.HomeScreenItemType_PEOPLE {
+			continue
+		}
+		people := item.Data.People()
+		ptyp, err := people.T()
+		if err != nil {
+			t.Fatal(err)
+		}
+		switch ptyp {
+		case keybase1.HomeScreenPeopleNotificationType_FOLLOWED:
+			follow := people.Followed()
+			if follow.User.Username == f {
+				return true, item.Badged
 			}
-			if typ == keybase1.HomeScreenPeopleNotificationType_FOLLOWED {
-				follow := people.Followed()
+		case keybase1.HomeScreenPeopleNotificationType_FOLLOWED_MULTI:
```

Comment:
Is this a bugfix?

Reply:
It only breaks for tests that have two follows in a row, and this PR has the first one.